src/main/webapp/resources/js/pages/projects/share/shareSlice.js

```diff
@@ -35,7 +38,35 @@ export const updateMoveSamples = createAction(
 );
 
 /**
- * Set up the initial state.  This is pulled from session storage which should
+ * Sets up the origainl restrictions to be the ame as what are on the source project.
```

Comment:
Typo in `original`

Reply:
Updated in fd72abd3f5a79c38196d95dc823feab7fa865a7e